src/common/meta/src/cache_invalidator.rs

```diff
@@ -137,6 +138,12 @@ where
                     let key = FlowInfoKey::new(*flow_id);
                     self.invalidate_key(&key.to_bytes()).await;
                 }
+                CacheIdent::FlowNode(node_id) => {
+                    // other caches doesn't need to be invalidated
+                    // since this is only for flownode address change not id change
+                    let key = NodeAddressKey::with_flownode(*node_id);
```

Comment:
please confirm is this the only key in kv backend cache that need to update for flownode's address?(Since others have a remap mechanism?) @WenyXu